storops/unity/resource/cg.py

```diff
@@ -179,17 +180,45 @@ def add_lun(self, *lun_list):
     def remove_lun(self, *lun_list):
         return self.modify(lun_remove=lun_list)
 
-    def replace_lun(self, *lun_list):
-        """Replaces the exiting LUNs to lun_list."""
-        existing_ids = {lun.get_id() for lun in self.luns}
-        new_ids = {lun.get_id() for lun in lun_list}
-        add_ids = new_ids - existing_ids
+    @property
+    def existing_lun_ids(self):
+        luns = self.luns if self.luns else []
+        return {lun.get_id() for lun in luns}
+
+    def _prepare_luns_add(self, lun_list):
+        if not lun_list:
+            return None
+        add_ids = {lun.get_id() for lun in lun_list} - self.existing_lun_ids
         log.debug("Adding LUN(s) '{}' to cg {}".format(add_ids, self.get_id()))
-        lun_add = [UnityLun(_id=_id, cli=self._cli) for _id in add_ids]
-        remove_ids = existing_ids - new_ids
+        return [UnityLun(_id=_id, cli=self._cli) for _id in add_ids]
+
+    def _prepare_luns_remove(self, lun_list, is_delta):
```

Comment:
Why not set the default value of is_delta to True

Reply:
I want to let the caller invoke this method with all arguments explicitly and be aware of the difference of passing in and not passing in `is_delta`.